#
# old_revision [efb32c6fbcef5b2251f98cc6e94ac9911090639a]
#
# patch "cbi-architecture/cbi-architecture.tex"
#  from [87cc6b2533b24728ce1327de5117e01fa8eb5a0a]
#    to [3c8ebe3837031e4dbba8abe0fb63c923f79d6eb1]
# 
# patch "cbi-scripting/cbi-scripting.tex"
#  from [bf6bafb927a4669b36d3bf0e392b05f2abe0259e]
#    to [78d1f052b002676bdd0ff0965af162bd21a34c8c]
#
============================================================
--- cbi-architecture/cbi-architecture.tex	87cc6b2533b24728ce1327de5117e01fa8eb5a0a
+++ cbi-architecture/cbi-architecture.tex	3c8ebe3837031e4dbba8abe0fb63c923f79d6eb1
@@ -2008,7 +2008,7 @@ \section*{Figure Legends}
 % Figure 1
 \begin{figure}[ht]
 \begin{center}
-\includegraphics[width=4in]{figures/exp-sim.eps}
+%\includegraphics[width=4in]{figures/exp-sim.eps}
 \end{center}
 \caption{
 {\bf Data flows in science.}
@@ -2022,7 +2022,7 @@ \section*{Figure Legends}
 % Figure 2
 \begin{figure}[ht]
 \begin{center}
-\includegraphics[width=4in]{figures/matrix.eps}
+%\includegraphics[width=4in]{figures/matrix.eps}
 \end{center}
 \caption{ {\bf Principle concerns.}  The four fundamental building
   blocks of a simulator are distinguished by separating (i) Data from
@@ -2041,9 +2041,9 @@ \section*{Figure Legends}
 % Figure 3
 \begin{figure}[ht]
 \begin{center}
-\includegraphics[width=4in]{figures/cbi-architecture-simple.eps}
+%\includegraphics[width=4in]{figures/cbi-architecture-simple.eps}
 \end{center}
-\caption{ {\bf A federated software architecture:} Graphical
+\caption{ {\bf A federated software architecture.} Graphical
   illustration of the different modules defined for the CBI
   federated software architecture.  Control modules are given to the left and data modules to the
   right.  A. The top layer contains conceptual data
